channels/email/accounts/account-setup.rst

```diff
@@ -143,10 +147,18 @@ SSL / STARTTLS
       * SSL
       * STARTTLS
 
+SSL Verification
```

Comment:
We tried to be consistent with the label capitalization, it should be `SSL verification` everywhere. Please let me know if you find a place where it's not like that.